src/java.base/share/classes/java/util/Arrays.java

```diff
@@ -4112,6 +4112,17 @@ public static boolean[] copyOfRange(boolean[] original, int from, int to) {
      * {@link Collections#unmodifiableList Collections.unmodifiableList}
      * or <a href="List.html#unmodifiable">Unmodifiable Lists</a>.
      *
+     * @implNote
+     * Modification methods of the returned list (such as {@link List#set} and
```

Comment:
(This got buried in a commit specific message that got lost with the updates.)

My personal preference would be to call out explicitly what happens here:
```
@implNote
The  {@link List} implementation returned by this method does not conform to the 
specification for List.replaceAll and List.set, in that in the event of encountering an element of an unsuitable type, ArrayStoreException will be thrown instead of ClassCastException.
```
(with suitable formatting of course). I don't think an example is necessary.